packages/nhost-js/src/clients/functions.ts

```diff
@@ -54,6 +55,19 @@ export class NhostFunctionsClient {
     })
   }
 
+  /** @deprecated Axios will be replaced by cross-fetch in the near future. Only the headers configuration will be kept. */
+  async call<T = unknown, D = any>(
+    url: string,
+    data?: D,
+    config?: (NhostFetchConfig | AxiosConfig) & { useAxios?: true }
+  ): Promise<DeprecatedFunctionCallResponse<T>>
+
+  async call<T = unknown, D = any>(
+    url: string,
+    data: D,
+    config?: NhostFetchConfig & { useAxios: false }
```

Comment:
```suggestion
    config?: NhostFunctionsConfig & { useAxios: false }
```

Reply:
`NhostFetchConfig` is also used in `nhost.graphql.request`, I guess you want to rename it there too?
Following that logic, we should rather name the two aliases `NhostFunctionCallConfig` and `NhostGraphQlRequestConfig` as they are not parameters of the clients, but of a method of the clients

Reply:
yes, I would also rename `NhostFetchConfig` in the `graphql` client. I can see how in the future we might have different options for functions vs graphql.

Reply:
> Following that logic, we should rather name the two aliases NhostFunctionCallConfig and

I am fine with that. In the same logic, `FunctionCallResponse` could become `NhostFunctionCallResponse`.